### DISCUSSION TITLE
How to get the Arctic Code Vault Contributor badge achievement?

### DISCUSSION THREAD

**Discussion by lincolnvc** [Schweinepriester/github-profile-achievements#42] [Q&A] [answered]
How to get the Arctic Code Vault Contributor badge achievement?

**Comment by Schweinepriester** [comment, marked as answer]
You can't anymore:

> Earnable?
> ❌

> Earned by
> Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)

> The 02/02/2020 snapshot archived in the GitHub Arctic Code Vault swept up every active public GitHub repository. It included every repo with any commits between the announcement at GitHub Universe on November 13th and 02/02/2020; every repo with at least 1 star and any commits from the year before the snapshot; and all repos with at least 250 stars. (It also included gh-pages for those repositories.)
